README.md

```diff
@@ -49,7 +49,7 @@ by this framework:
 | [`cJSON` ](https://github.com/DaveGamble/cJSON/issues/800) | Vertex AI | [default](prompts/template_xml) |
 | [`libplist`](https://github.com/libimobiledevice/libplist/issues/244) | Vertex AI | [default](prompts/template_xml) |
 | [`hunspell`](https://github.com/hunspell/hunspell/issues/996) | Vertex AI | [default](prompts/template_xml) |
-
+| [Undisclosed stack buffer underflow](https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=67483) | Vertex AI|  [default](prompts/template_xml) |
```

Comment:
nit: preserve the empty line to visually separate two sections when reading the markdown?

Reply:
done!

Reply:
Thanks!